content/post/21_05_07-progress-report-26.md

```diff
@@ -0,0 +1,241 @@
+
+### Context Menu Improvements
+*Contributed by daniel-cr*
+
+{{< imgw "/images/post/pr_26/context" >}}
+
+The context menu has been improved, making large stacks of items easier to manage.
+
+### Votes
+*Contributed by PJB3005*
+
+Players can now vote to restart the round (i ded) or change the next round's gamemode.
+
+### Better Admin Panel
+*Contributed by lzimann, DrSmugleaf and metalgearsloth*
+
+{{< imgw "/images/post/pr_26/admin" >}}
+
+You can now kick, ban and teleport to players from the admin panel.
+
+### Health Overlay
+*Contributed by DrSmugleaf*
+
+<video src="/video/pr_26/health.mp4" playsinline></video>
+
+A simple command which displays health-bars over players heads. Useful for testing and admins.
+
+## Tiny things
+- Holiday system added. The game can now run special events on certain dates and give messages like "Merry Christmas" *Contributed by Zumorica*
+- Gas leak event *Contributed by metalgearsloth*
+- Inflatable walls *Contributed by ShadowCommander*
+- Stacked sprites now can be visualized *Contributed by Ygg01*
+- Admin menu now can set temperature and add gases *Contributed by metalgearsloth*
+- Extended gamemode *Contributed by Zumorica in less than 2:30 minutes*
+- You can customize your eyes color now *Contributed by Rember*
+- Unified the server/client consoles *Contributed by Acruid*
+- You now spill whatever is in your hand when falling. *Contributed by Clyybber* <!-- We need a video of someone falling with a potassium and water beakers in hand -->
+- Latejoins are now announced via radio *Contributed by ike709*
+- Verb icons *Contributed by Swept*
+- Ghosts can now spook players by flickering lights *Contributed by Macoron*
+
+## Technical details
+
+### Fluent Localization
+*Contributed by RemieRichards*
+
+### Lighting Optimizations
+*Contributed by PJB3005*
+
+### Physics rework
+*Contributed by metalgearsloth*
+
+Our previous physics iterations were notoriously unstable so we went to the effort of integrating Box2D into the engine (specifically basing it off of the Farseer dotnet port).
+It is by no means finished yet but the features currently include:
+* Circle colliders for players
+* Items now have collision without tanking server performance
+* Weightlessness rubberbanding is fixed
+* Pushing is less jittery
+* Running along walls is slightly less jittery
+* Throwing is more realistic
+* Preliminary work for performantly driving the station
+
+We'll be doing work to make the physics system more performant so it will continue to get faster over time.
+
+This also allows us to remove a lot of cruft with controllers as previously velocity was being reset every frame so if you wanted a body to continually move you'd have to manually push it; now you can just apply an impulse once and the system will take care of it.
+
+### Serialization v3
+*Contributed by PaulRitter and DrSmugleaf*
+<!-- Documentation: https://hackmd.io/@ss14/serialization -->
```

Comment:
commenting this just in case i do not manage to write something for this.
what should be mentioned:
- all serialization is being done through datadefinitions
- yaml linting(!)
- this opens up the possibility of a yaml/prototype editor
- more ease of use yada yada you get the idea